docs/preview/02-Features/powershell/azure-integration-account.md

```diff
@@ -347,4 +351,210 @@ An example of this file:
+
+**Example**  
+
+Uploading a *single agreement* into an Integration Account.  
+```powershell
+PS> Set-AzIntegrationAccountAgreements -ResourceGroupName 'my-resource-group' -Name 'my-integration-account' -AgreementFilePath "C:\Agreements\MyAgreement.json"
+# Uploading agreement 'MyAgreement' into the Azure Integration Account 'my-integration-account'
+# Agreement 'MyAgreement' has been uploaded into the Azure Integration Account 'my-integration-account'
+```
+
+Uploading a *single agreement* into an Integration Account and set add a prefix to the name of the agreement within the Integration Account.  
+```powershell
+PS> Set-AzIntegrationAccountAgreements -ResourceGroupName 'my-resource-group' -Name 'my-integration-account' -AgreementFilePath "C:\Agreements\MyAgreement.json" -ArtifactsPrefix 'dev-'
+# Uploading agreement 'dev-MyAgreement' into the Azure Integration Account 'my-integration-account'
+# Agreement 'dev-MyAgreement' has been uploaded into the Azure Integration Account 'my-integration-account'
+```
+
+Uploading *all agreements* located in a specific folder into an Integration Account.  
+```powershell
+PS> Set-AzIntegrationAccountAgreements -ResourceGroupName 'my-resource-group' -Name 'my-integration-account' -AgreementsFolder "C:\Agreements"
+# Uploading agreement 'MyFirstAgreement' into the Azure Integration Account 'my-integration-account'
+# Agreement 'MyFirstAgreement' has been uploaded into the Azure Integration Account 'my-integration-account'
+# ----------
+# Uploading agreement 'MySecondAgreement' into the Azure Integration Account 'my-integration-account'
+# Agreement 'MySecondAgreement' has been uploaded into the Azure Integration Account 'my-integration-account'
+# ----------
+```
+
+Uploading *all agreements* located in a specific folder into an Integration Account and set add a prefix to the name of the agreements.
+```powershell
+PS> Set-AzIntegrationAccountAgreements -ResourceGroupName 'my-resource-group' -Name 'my-integration-account' -AgreementsFolder "C:\Agreements" -ArtifactsPrefix 'dev-'
+# Uploading agreement 'dev-MyFirstAgreement' into the Azure Integration Account 'my-integration-account'
+# Agreement 'dev-MyFirstAgreement' has been uploaded into the Azure Integration Account 'my-integration-account'
+# ----------
+# Uploading agreement 'dev-MySecondAgreement' into the Azure Integration Account 'my-integration-account'
+# Agreement 'dev-MySecondAgreement' has been uploaded into the Azure Integration Account 'my-integration-account'
+# ----------
+```
+
+**Agreement JSON Example**
+The agreement definition is the JSON representation of your agreement, this JSON definition can also be viewed in the Azure Portal using https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-agreements#edit-an-agreement and clicking on `Edit as JSON`.
+An example of this file:
+```
```

Comment:
We could maybe see with @Mcalewaert , if there's a way to 'wrap' this bc it can be rather tedious to view this by default on the page.

Reply:
dropdown has been implemented 😎